.github/workflows/build-containers.yml

```diff
@@ -20,7 +20,7 @@ on:
     types: [published]
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.run_number }}
+  group: build_containers--${{github.ref}}-${{github.run_number}}
```

Comment:
The name before was designed to:
1. Avoid skipping commits on `develop`, i.e. if we merged multiple PRs in a short amount of time each commit would be tested
2. Cancel runs on PRs if a newer commit is being pushed while a workflow is running

It seems to me that always using `run_number` we are losing 2, or am I missing something?

Reply:
Crud, you are probably right, I can put the conditional pull_request number back but keep the ref and prefixes.